qa/tasks/cephadm.py

```diff
@@ -118,14 +113,19 @@ def download_cephadm(ctx, config, ref):
                     'tar', '-xO', 'src/cephadm/cephadm',
                     run.Raw('>'),
                     ctx.cephadm,
-                    run.Raw('&&'),
-                    'test', '-s',
-                    ctx.cephadm,
-                    run.Raw('&&'),
-                    'chmod', '+x',
-                    ctx.cephadm,
                 ],
             )
+        # sanity-check the resulting file and set executable bit
+        cephadm_file_size = '$(stat -c%s {})'.format(ctx.cephadm)
+        ctx.cluster.run(
+            args=[
+                'test', '-s', ctx.cephadm,
+                run.Raw('&&'),
+                'test', run.Raw(cephadm_file_size), "-gt", run.Raw(1000),
+                run.Raw('&&'),
+                'chmod', '+x', ctx.cephadm,
+            ],
+        )
```

Comment:
Can this be replace with more human readable code? Like:
```
cox.cluster.run(args="test -s {path} && 
                      test $(stat -c%s {path}) -g 1000 && 
                      chmod +x {path}".format(path=ctx.cephadm))
```

Reply:
well, I was trying to emulate the coding style that is being used immediately before and after this section...

maybe it would be better to open a new PR that refactors all of the `remote.run` calls to make them more readable

Reply:
ok, that was just a suggestion

Reply:
so you think we need to wait until this new code merged and then fix it?

Reply:
I think if we want to change code that looks like this:

```
remote.run(args=[ 'ls', '-l', 'foo', run.Raw('>'), 'bar' ])
```

to this:

```
remote.run(args="ls -l foo > bar")
```

then we should do it all at once throughout the code base, not piecemeal